frontend/src/commons/Header.js

```diff
@@ -1,23 +1,39 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
 import styled from 'styled-components/macro';
-import { FaPlus } from 'react-icons/fa';
+import { CgAddR } from 'react-icons/cg';
 import { IoMdLogOut } from 'react-icons/io';
 import Navbar from './Navbar';
 import NavItem from './NavItem';
 import AddDropdownMenu from './AddDropdownMenu';
+import UserContext from '../contexts/UserContext';
 
 export default function Header({ title }) {
+  const history = useHistory();
+  const { logout } = useContext(UserContext);
+
   return (
     <HeaderStyled>
-      <HeadingStyled>{title}</HeadingStyled>
-      <Navbar>
-        <NavItem icon={<FaPlus />}>
-          <AddDropdownMenu></AddDropdownMenu>
-        </NavItem>
-        <NavItem icon={<IoMdLogOut />} />
-      </Navbar>
+      <Heading>{title}</Heading>
+      {window.location.pathname === '/favorites' ? (
```

Comment:
Better control this via a prop, maybe `showDropdown` that has a default value of `false`, and set it accordingly in when on `/favorites`.